modules/ROOT/pages/genai-integrations.adoc

```diff
@@ -266,19 +266,56 @@ Supported values:
 
 | model
 | STRING
-| The ID of the model you want to invoke. +
-Supported values: `"text-embedding-ada-002"`
-| "text-embedding-ada-002"
+| The name of the model you want to invoke.
+| `"text-embedding-ada-002"`
+
+| dimensions
+| INTEGER
+| The number of dimensions you want to reduce the vector to. Only supported for certain models (see official documentation).
+| Model-dependent.
+|===
+
+=== Azure OpenAI
```

Comment:
This will add a label aligned with the header specifying the release version.
```suggestion
[role=label--new-5.18]
=== Azure OpenAI
```

Reply:
Well we may have to be careful here, should it be `5.18` (Neo4j version its packaged with) or `1.1.0` (the plugin version).
This is something we've been debating in other areas.

Reply:
Ok - I can't speak to the plugin version, but if a user can only use it on 5.18 and later, then this should be added. Could we mention the plugin version details in the main body of the text?

Reply:
the plugin _was_ independent of the neo4j version; we've since added some private metrics which _does_ effectively tie it to a minimum neo4j version; but I believe we've just noted that it is from Neo4j 5.17 as a 'release' (despite actually being packaged earlier)

Reply:
Since I think the conclusion from the other discussions was that it's reasonable to make the plugin version the same as the Neo4j version (but hide it in Aura), I think this label makes sense.